Pipfile

```diff
@@ -22,6 +22,7 @@ marshmallow = "==3.20.1"
 networkx = "==2.6"
 openpyxl = "==3.1.2"
 pandas = "==1.3.5"
+numpy = "<2.0.0"
```

Comment:
Should we put this under "dev-packages"?
Just to indicate that this condition is only used when developing taipy, not an actual taipy requirement.

Reply:
I like it. Changing it now